### ISSUE TITLE
Skyrim special edition support

### ISSUE THREAD

**Issue by Utumno** [wrye-bash/wrye-bash#347]
Post small fixups and ideas for skyrim SE support here

**Comment by Utumno** [comment]
I pushed a final version of the skyrim-se-support branch. I need someone to verify that:

- constants are alright. What I do is import the skyrim ones and remove what's removed and add what's new. I may have missed stuff
- records are alright. Sharlikran decided to again start working on his own and has copy pasted the entire skyrim.records and started modifying that. I need someone to verify all needed changes are in the skyrimse.records.

That's very important cause next step is merging to dev

**Comment by Arthmoor** [comment]
The constants all look fine.

Can't comment on the records though. That's outside my knowledge.

**Comment by Utumno** [comment]
Ok I will dump them in and let the users test it then - it's a beta after all and it's not my area of expertise either.

Will just notify you here so you test the final packaging - but hey may take longer as loot 10 seems ready to ship...

@Scrivener07: I see you thumbed up a couple posts (including a technical one) - any chance you lend a hand ? :P

**Comment by Utumno** [comment]
TODOs:

- reading save masters needs lz4 support - blocked by #338
- installer needs fixups
- revisit records

**Comment by fireundubh** [comment]
Where is the uninstaller? Is it in the repo?

**Comment by Utumno** [comment]
All the installer scripts are in `\scripts\build\installer\*.nsi` files

**Comment by fireundubh** [comment]
Okay, I think I fixed the uninstaller, but I don't know how to test the fixed scripts.

https://github.com/fireundubh/wrye-bash/commit/d83ebbe1fe2b8616195f74adc150cb92a17821ef

**Comment by Utumno** [comment]
Thanks, really appreciated :)

Please pack this and post to the beth thread so regulars (in particular alt3rn1ty aka w0nkyd0nky) can test

The way to pack is package_for_release.py - have a look at https://github.com/wrye-bash/wrye-bash/wiki/[github]-Making-A-Release - you only need a couple of steps the rest are for updating post and the like

**Comment by fireundubh** [comment]
[Posted.](http://forums.bethsoft.com/topic/1612572-wrye-bash-thread-112/?p=25277474)

I tested in [Sandboxie](https://www.sandboxie.com/). Everything seems to be cleaned up properly.

**Comment by fireundubh** [comment]
I fixed the issue with resources and shortcuts not installing in this branch:

https://github.com/fireundubh/wrye-bash/commits/nsis-x64-fix

The problem was that several variables weren't declared properly or were missing the relevant prefixes (e,g., CheckState, Path), so the resources and shortcuts installers weren't running at all.

**Comment by Utumno** [comment]
Thanks a lot @fireundubh :)

Yes basically all I wanted was the installer functioning correctly - don't pay attention to new features stuff. If interested in iaz3 work it's here: https://github.com/iaz3/wrye-bash/commits/iaz3-pyinstaller-wip but I haven't really looked at it at all

Will have a look at the branch asap but meanwhile you could offer it to testers

**Comment by fireundubh** [comment]
Fixed another issue where Wrye Bash could possibly not be launched from the final page of the installer for some games (TES5, SSE, and FO4) due to a typo in those path variables.
